content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+In machine learning projects, number of <abbr>experiments</abbr> grow rapidly.
+DVC can track these experiments, list their most relevant parameters and
```

Comment:
it helps to run them also, navigate, compare ...

Reply:
Added. Thanks.